README.md

```diff
@@ -14,12 +14,12 @@ This repo provides a simple CLI tool to run games locally against your dev envir
 Download precompiled binaries here: <br>
 [https://github.com/BattlesnakeOfficial/rules/releases](https://github.com/BattlesnakeOfficial/rules/releases)
 
-Install as a Go package. Requires Go 1.17 or higher. [[Download](https://golang.org/dl/)]
+Install as a Go package. Requires Go 1.18 or higher. [[Download](https://golang.org/dl/)]
```

Comment:
Based on the docs for the `go` setting in `go.mod`, I don't think this is actually true. In order to find out the required Go version, we'd need to double check what language features we're using.
We're likely going to use new features like fuzzing and generics at some point though, so this version will become required.